test/cbmc/proofs/Routing/MatchingEndpoint/Makefile.json

```diff
@@ -0,0 +1,17 @@
+{
+    "ENTRY": "MatchingEndpoint",
+    "CBMCFLAGS":
+    [
+      "--unwindset pxEasyFit.0:5",
```

Comment:
Can we set a macro to describe what 5 is?

Reply:
Like [Aniruddha commented](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/896#discussion_r1230101287) in another PR.